modmail/bot.py

```diff
@@ -124,3 +231,4 @@ def remove_cog(self, cog: str) -> None:
     async def on_ready(self) -> None:
         """Send basic login success message."""
         self.logger.info("Logged in as %s", self.user)
+        await self.change_presence(status=discord.Status.online, activity=self.activity)
```

Comment:
Don't put change_presence in on_ready, as on_ready may be fired weirdly. Bot init takes a status argument

Reply:
On_Ready